content/tutorials/quickstart/py-batch-minting.md

```diff
@@ -0,0 +1,405 @@
+---
+html: py-batch-minting.html
+parent: quickstart-python.html
+blurb: Mint multiple NFTs with the press of a button.
+labels:
+  - Accounts
+  - Quickstart
+  - NFTs
+  - XRP
+---
+
+# Batch Mint NFTs
+
+You can create an application that mints multiple NFTs at one time, using a `for` loop to send one transaction after another.
+
+A best practice is to use `Tickets` to reserve the transaction sequence numbers. If you create an application that creates NFTs without using tickets, if any transaction fails for any reason, the application stops with an error. If you use tickets, the application continues to send transactions, and you can look into the reason for any individual failures afterward.
+
+[![Batch Mint](img/quickstart-py36.png)](img/quickstart-py36.png)
+
+## Usage
+
+You can download or clone the [Quickstart Samples](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples/quickstart/py/){.github-code-download} to try the sample in your own browser.
+
+## Get an Account
+
+1. Open and run `lesson7-batch-minting.py`.
+2. Get a test account.
+    1. If you want to use an existing account seed:
+        1. Paste the account seed in the **Standby Seed** field.
+        2. Click **Get Standby Account**.
+    2. If you do not want to use an existing account seed, click **Get Standby Account**.
+
+
+## Batch Mint NFTs
+
+This example lets you mint multiple NFTs for a single unique item. The NFT might represent "prints" of an original artwork, tickets to an event, or another limited set of unique items. 
+
+To batch mint a non-fungible token objects:
+
+1. Set the **Flags** field. For testing purposes, we recommend setting the value to _8_. This sets the _tsTransferable_ flag, meaning that the NFT object can be transferred to another account. Otherwise, the NFT object can only be transferred back to the issuing account. See [NFTokenMint](nftokenmint.html) for information about all of the available flags for minting NFTs.
+2. Enter the **NFT URI**. This is a URI that points to the data or metadata associated with the NFT object. You can use the sample URI provided if you do not have one of your own.
+3. Enter an **NFT Count** of up to 200 NFTs to create in one batch.
+4. Enter the **Transfer Fee**, a percentage of the proceeds that the original creator receives from future sales of the NFT. This is a value of 0-50000 inclusive, allowing transfer fees between 0.000% and 50.000% in increments of 0.001%. If you do not set the **Flags** field to allow the NFT to be transferrable, set this field to 0.
```

Comment:
Do we have to specify .001% increments if we already specified the decimal precision as 0.000% to 50.000%?

Reply:
Probably not, but I don't think it hurts to do so.